libmattermost.c

```diff
@@ -2226,6 +2226,34 @@ mm_process_attachment(JsonObject *attachment)
 	return message;
 }
 
+static gint64
+mm_get_channel_approximate_view_time(MattermostAccount *ma, gchar *id)
+{
+	gchar *tmptime = NULL;
+
+	PurpleChat *chat = mm_purple_blist_find_chat(ma, id);
+	if (chat) {
+		tmptime = g_strdup(purple_blist_node_get_string(PURPLE_BLIST_NODE(chat),"channel_approximate_view_time"));
+	} else {
+		PurpleBuddy *buddy = purple_blist_find_buddy(ma->account, g_hash_table_lookup(ma->one_to_ones,id));
+		if (buddy) {
+			tmptime = g_strdup(purple_blist_node_get_string(PURPLE_BLIST_NODE(buddy),"channel_approximate_view_time"));

```

Comment:
Also use constant for `channel_approximate_view_time`.
```suggestion
			tmptime = g_strdup(purple_blist_node_get_string(PURPLE_BLIST_NODE(buddy), "channel_approximate_view_time"));

```

Reply:
Have not used constant for any existing code as per this https://github.com/Brightscout/purple-mattermost/pull/3#discussion_r825707728